.gitignore

```diff
@@ -70,4 +70,11 @@ docs/_build/
 /external/libjpeg-turbo
 /external/pthreads
 /external/miniconda*
-Miniconda*.exe
\ No newline at end of file
+Miniconda*.exe
+# build folders
+jasper-version-2.0.16
```

Comment:
Could you do this in a separate PR? I think a better solution would be to change the mac build script so that everything is cloned/downloaded into external/. And then I would use similar ignore patterns like for the other folders in external/, avoiding hard-coding of version numbers. Like `/external/jasper*`.

Reply:
Ok sounds good.
How about I do this in #108 ?

Reply:
Yes, sounds good.